react/ShelfContent.js

```diff
@@ -84,17 +85,18 @@ class ShelfContent extends Component {
   }
 
   ssrFallback() {
-    const { products, itemsPerPage } = this.props
-    const className = this.getClassByItemsPerPage(itemsPerPage)
+    const { products, itemsPerPage, isMobile } = this.props
+    const items = isMobile ? DEFAULT_ITEMS_MOBILE : itemsPerPage
```

Comment:
It's not better do rename this `const` value to `numberOfItems` or something like that? The term items is better used when you are assigning an array of items, don't you think?